tests/update/patch_before_migration.pm

```diff
@@ -73,6 +79,53 @@ sub patching_sle {
     set_var("SYSTEM_PATCHED", 1);
 }
 
+sub install_package {
+    my @pk_list = split(/,/, get_var('PATTERNS'));
```

Comment:
`PACKAGES`?

Reply:
add this part by @QingmingSu 's Comments, in case we need install some packages for modules which don't have pattern.

Reply:
You try to read packages list from PATTERNS, seems incorrect

Reply:
Thanks for that, I have rebase it.